src/components/v5/common/ActionSidebar/hooks/useGetActionData.ts

```diff
@@ -42,7 +42,7 @@ const useGetActionData = (transactionId: string | undefined) => {
     amounts,
     refetchStreamingPayment,
     updateAmountsAndStatus,
-  } = useGetStreamingPaymentData(action?.expenditureId);
+  } = useGetStreamingPaymentData(action?.streamingPaymentId);
```

Comment:
Inside this hook the skip condition is `Number.isNaN` which is false for `null` or empty string so there's a bunch of requests without ID in the network tab. Could you change that as well?